pytorch_lightning/core/lightning.py

```diff
@@ -1519,10 +1521,20 @@ def __init__(self, hparams):
             checkpoint = torch.load(checkpoint_path, map_location=lambda storage, loc: storage)
 
         if tags_csv is not None:
-            # add the hparams from csv file to checkpoint
-            hparams = load_hparams_from_tags_csv(tags_csv)
-            hparams.__setattr__('on_gpu', False)
-            checkpoint['hparams'] = vars(hparams)
+            hparams_file = tags_csv
+            warnings.warn('tags_csv argument is deprecated, use hparams_file argument instead.')
```

Comment:
uze `zero_rand_warn` and specify from when it is deprecated and till when
```suggestion
            rank_zero_warn('`tags_csv` argument is deprecated in v0.7.6. Will be removed v0.9.0', DeprecationWarning)
```